pages/contact.tsx

```diff
@@ -154,49 +159,47 @@ const Contact: NextPage = () => {
       </div>
       <div className="relative -top-32 sm:relative sm:-top-64 xl:-top-48 2xl:-top-80">
         <div className="w-full h-screen   flex justify-center items-center z-50">
-          <div className="w-5/4 sm:w-3/5   sm:p-10 bg-white max-h-fit overflow-hidden border-2 rounded-2xl shadow-md hover:shadow-none">
+          <div className="w-4/5 sm:w-4/5   sm:p-10 bg-white max-h-fit overflow-hidden border-2 rounded-2xl shadow-md hover:shadow-none">
             <div className="pl-5 pr-5 mb-3">
-              <h1 className="text-3xl font-bold">Lets Connect</h1>
+              <h1 className="text-3xl font-bold">
+                {TEXT_CONSTANTS.LEST_CONNECT}
+              </h1>
               <h1 className="text-xs max-w-xs mt-2">
-                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-                eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                {TEXT_CONSTANTS.GET_IN_TOUCH}
               </h1>
             </div>
 
             <div className="flex flex-col justify-center items-center space-x-5 sm:flex-col md:flex-row">
               <div className="w-full sm:w-full md:w-3/4 ">
                 {/* form start */}
                 <div>
-                  <form onSubmit={handleSubmit}>
+                  <form id="registrationForm" onSubmit={handleSubmit}>
                     <div className="flex flex-col  md:space-x-2 mb-3 justify-center rounded-md pl-5 pr-5 md:flex-row sm:flex-col">
                       <div className="w-full">
                         <label className="block p-2 xl:text-lg 2xl:text-2xl">
                           Full Name
```

Comment:
Shift label input and error to a reusable component, also render the fields from a json object instead of current approach